django/contrib/postgres/fields/ranges.py

```diff
@@ -227,6 +236,7 @@ def get_prep_lookup(self):
 models.IntegerField.register_lookup(RangeContainedBy)
 models.BigIntegerField.register_lookup(RangeContainedBy)
 models.FloatField.register_lookup(RangeContainedBy)
+models.DecimalField.register_lookup(RangeContainedBy)
```

Comment:
Don't we still need to add `models.SmallIntegerField.register_lookup(RangeContainedBy)` here?

Reply:
Since everything inherits from `IntegerField` it should be enough to only register lookup for it. In short `models.BigIntegerField.register_lookup(RangeContainedBy)` is redundant.